Add Nothing AG to list of contributors in the README

renestalder opened the pull request:
Going to make a first small step and adding Nothing AG to the list of contributors in the README. Will check the for the logo addition to the guide later.

Commit:
docs(readme): add Nothing AG to list of contributors

renestalder commented:
@gurix Is this okay?

gurix commented:
@renestalder For me it is alright, but the final decision is made during the review I guess. I changed the title of this PR to WIP and suggest you also add the logos here. So we can review later everything together. Alright?

renestalder commented:
:+1:

AndreasMoesch commented:
@rey attaching the logo image file:

![nothing_logowithtype](https://user-images.githubusercontent.com/9501560/49690966-57d3b500-fb39-11e8-85ac-5847bab64ad2.png)

I will freely admit that the colour contrast of the sharp orange against the green background is very unflattering (not to mention also inaccessible). 

## Startpage
(quickly replaced the HF logo there; obviously, in the end it would be 5 logos)
![screen shot 2018-12-08 at 22 24 43 2](https://user-images.githubusercontent.com/9501560/49691061-ce24e700-fb3a-11e8-85fc-a7d3449519c8.png)

## “About” section
![screen shot 2018-12-08 at 22 28 09 2](https://user-images.githubusercontent.com/9501560/49691066-ee54a600-fb3a-11e8-8543-21b5f5c0272c.png)

---

I also tried it with [just the logo](https://user-images.githubusercontent.com/9501560/49691076-1e03ae00-fb3b-11e8-8012-1bbb707f1d87.png), leaving aside the “nothing” word mark. 

It looked somewhat clunky to me on the 
[startpage](https://user-images.githubusercontent.com/9501560/49691094-6753fd80-fb3b-11e8-93f8-764ba4044fae.png) and [in the “About” section](https://user-images.githubusercontent.com/9501560/49691097-7a66cd80-fb3b-11e8-94ac-e6a15768cc8a.png)

gurix commented:
@AndreasMoesch True, the orange is not going well together with the green background. Perhaps a white background square around the logo could solve this issue? On the `About` Page i really like the logo with the text.

AndreasMoesch commented:
@gurix Good input! I tried a white background with the orange logo and an orange background with the white logo. I also made the “nothing” word bolder so that it’s almost legible on this size.

![screen shot 2018-12-16 at 22 47 27](https://user-images.githubusercontent.com/9501560/50059294-4f0c6000-0185-11e9-96aa-fd8ff28f5075.png)

Personally, I’d prefer the orange background but depending on your browser that orange would overpower everything. That logo should not be the most eye-catching thing on this page 

**—> So I’d go with the white background instead**

Regarding colour rendering: Some browser comparisons

![screen shot 2018-12-16 at 22 51 40](https://user-images.githubusercontent.com/9501560/50059298-559ad780-0185-11e9-9aa3-552957fca1ee.png)

![screen shot 2018-12-16 at 22 57 58](https://user-images.githubusercontent.com/9501560/50059374-47998680-0186-11e9-9b8f-80f4eba021e0.png)

AndreasMoesch commented:
Attaching special logo version for small views

## ⚠️ Please don’t use this anywhere else than on the start page ⚠️ 

![nothing_logowithtype-whitetransparentbg-bolder](https://user-images.githubusercontent.com/9501560/50059404-a52dd300-0186-11e9-80c3-718d858d89fb.png)

Commit:
Merge branch 'upstream/1.1' into docs/readme

Commit:
feat: add Nothing AG logo to footer

Commit:
refactor: rename nothing footer logo asset

Commit:
feat(About): add Nothing as a contributing web agency

Commit:
fix(footer): fix Nothing logo reference in background image

Commit:
fix(About): fix link to Nothing website

Commit:
refactor(About): retext part of Nothing's quote

renestalder commented:
* Updated the footer with the Logo from Nothing. Had to make all logos some pixel smaller to make them fit.
* Added a quotation from Nothing to the about page.
* I didn't make changes to the list of the core team as the photo wouldn't fit the list of members. Will propose our addition with the change in #194.

*Footer*:

<img width="695" alt="screenshot 2019-01-31 at 16 51 30" src="https://user-images.githubusercontent.com/105358/52066399-c3099880-2578-11e9-8370-92fa38edbae1.png">

*About page*:

<img width="695" alt="screenshot 2019-01-31 at 16 51 45" src="https://user-images.githubusercontent.com/105358/52066404-c735b600-2578-11e9-830f-df2c82d04367.png">

renestalder commented:
Side note: The order of the logo in the footer was chosen by what I felt is visually the best. I took the same order to place our description on the about page. Let me know in case this is a problem for someone. I don't want that someone feels like we tried to push ourselves higher into the list.

jmuheim commented:
Well done.

I assume this is also already merged into master branch, right?

renestalder commented:
@jmuheim Correct. That went to release 1.1 and was then cherry picked into master.

jmuheim commented:
Perfect. 👍
